tests/test_admin_serializers.py

```diff
@@ -0,0 +1,47 @@
+from unittest.mock import MagicMock, patch, sentinel
+
+from sfdo_template_helpers.admin.serializers import AdminAPISerializer
+
+
+class TestAdminAPISerializer:
+    def test_build_url_field(self):
+        class SubclassedAdminAPISerializer(AdminAPISerializer):
+            class Meta:
+                class model:
+                    class _meta:
+                        object_name = "Foo"
```

Comment:
📐

Reply:
While I appreciate the feedback, I can't say that I understand it.

Reply:
oh it was just an amusing amount of single indentation, I'm glad to see this going in :+1: